example/screens/CallbacksExample.tsx

```diff
@@ -34,13 +34,7 @@ export default function CallbacksExample() {
     return () => {
       Maps.removeAllListeners();
     };
-  }, [
-    clickSub,
-    dragStartSub,
-    dragEndSub,
-    cameraMoveEndSub,
-    cameraMoveStartSub,
-  ]);
```

Comment:
Why do we want to remove those as dependencies of `useEffect`?

Reply:
My bad, it should be left as it was